core/build.gradle

```diff
@@ -48,6 +49,7 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'
     testImplementation 'org.springframework.boot:spring-boot-starter-test:2.7.10'
     testImplementation 'com.squareup.okhttp3:mockwebserver:4.11.0'
+    testImplementation 'com.github.seregamorph:hamcrest-more-matchers:0.1'
```

Comment:
Adds some useful matchers, specifically `where` which is useful in combination with `everyItem` for running a check against every item in a collection.